### ISSUE TITLE
Tooltips don't announce

### ISSUE THREAD

**Issue by helenezspeer** [bootstrap-vue/bootstrap-vue#6565]
### Describe the bug

On Firefox with NVDA on Windows, tooltips appear on focus but NVDA doesn't announce the tooltip content.

### Steps to reproduce the bug

1. Launch Firefox with NVDA
2. Go to https://bootstrap-vue.org/docs/components/tooltip#b-tooltip-component-usage
3. Tab to Live Chat button
4. See tooltip appear; do not hear it

However...

1. Launch Firefox with NVDA
2. Go to https://bootstrap-vue.org/docs/components/tooltip#b-tooltip-component-usage
3. **Use the mouse to hover Live Chat button**
4. Tab to Live Chat button
5. Hear button text and tooltip text

Is it possible that the tooltip text isn't reading because the tooltip isn't in the DOM when the button is first focused?

### Expected behavior

Upon focus of the button, I expected to hear "Live Chat button Online"; instead I heard "Live Chat button".

**Environment:**

- Device: Mac
- OS: Windows 10
- Browser: Firefox
- Version: [e.g. 70]
